ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -725,6 +725,19 @@ tests:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-ipi-proxy-dualstack-privmaryv6-f28-disasterrecovery
+  cron: 7 1 3 * *
```

Comment:
Can we add test frequency for it? at least once a week.

Reply:
Updated.